oxo.py

```diff
@@ -0,0 +1,133 @@
+class OxoBoard:
+    def __init__(self, x, y):
+

```

Comment:
Whilst putting blank lines between `def` and the function body is not forbidden by PEP-8, it does make the code a little difficult to read.